persist/adapter_ex.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 The casbin Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package persist
+
+import (
+	"github.com/casbin/casbin/v2/model"
+)
+
+// AdapterEX is the interface for Casbin adapters with multiple add and remove policy functions.
+type AdapterEx interface {
```

Comment:
AdapterEx should extend Adapter. Do not make another copy for Adapter functions.